schema/ingestion_config/migrations/issue_997.py

```diff
@@ -0,0 +1,185 @@
+"""
+This script migrates the ingest config files to a new schema based on the changes outlines in
+https://github.com/chanzuckerberg/cryoet-data-portal/issues/997
+"""
+import glob
+import itertools
+import json
+import traceback
+from typing import Union
+
+import numpy as np
+import yaml
+
+
+def rawtilts_to_collection_metadata(config: dict) -> None:
+    list_globs = []
+    if 'rawtilts' in config:
+        for i in config['rawtilts']:
+            if "sources" not in i:
+                continue
+            old_source = i["sources"][0]["source_multi_glob"]["list_globs"]
+            list_globs.extend(s for s in old_source if s.endswith('.mdoc') or "{mdoc_name}" in s)
```

Comment:
The check for `"{mdoc_name}"` might not catch all the formatted strings in the config, as they would have been manually added.

Reply:
can you elaborate?

Reply:
This was just a heads up that the manually generated configs could use names that might not necessarily be consistent with `{mdoc_name}`, so for configs outside the gjensen folder, we should ensure that this worked.